cloudant-client/src/test/java/com/cloudant/tests/AttachmentsTest.java

```diff
@@ -241,26 +221,42 @@ public void attachmentStandaloneGivenId() throws IOException, URISyntaxException
         }
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void attachmentStandaloneNullDocNonNullRev() {
-        byte[] bytesToDB = "binary data".getBytes();
-        ByteArrayInputStream bytesIn = new ByteArrayInputStream(bytesToDB);
-        Response response = db.saveAttachment(bytesIn, "foo.txt", "text/plain", null, "1-abcdef");
+        assertThrows(IllegalArgumentException.class, new Executable() {
+            @Override
+            public void execute() throws Throwable {
+                byte[] bytesToDB = "binary data".getBytes();
+                ByteArrayInputStream bytesIn = new ByteArrayInputStream(bytesToDB);
+                Response response = db.saveAttachment(bytesIn, "foo.txt", "text/plain", null,
+                        "1-abcdef");
+            }
```

Comment:
Can we consider moving the code in the block above into a helper util method since it occurs several times in this test class?

Reply:
Probably outside the scope of what I'm doing, which is just translating the tests to junit5 without considering too closely how these tests are written.